client/src/pages/ManageBookings/ManageBookings.tsx

```diff
@@ -0,0 +1,141 @@
+import { useState } from 'react';
+import PageContainer from '../../components/PageContainer/PageContainer';
+import clsx from 'clsx';
+import { Box, Grid, Typography, Avatar } from '@mui/material';
+import useStyles from './useStyles';
+import CardWrapper from '../../components/CardWrapper/CardWrapper';
+import SettingsIcon from '@mui/icons-material/Settings';
+import { NavLink } from 'react-router-dom';
+import dogPicture from '../../images/landing/hero.jpg';
+
+import MyCalendar from '../../components/MyCalendar/MyCalendar';
+
+const RequestInfo: React.FC<{
+  date?: string | Date;
+  avatar?: any;
+  name?: string;
+  status?: string;
+  size?: any;
+}> = ({ date, avatar, name, status, size }) => {
+  const classes = useStyles();
+
+  return (
+    <Box className={clsx(classes.requestInfo, size === 'large' && classes.requestInfoLarge)}>
```

Comment:
We can use the MUI Card API to add semantic meaning - as each booking card contains info about a particular booking

Ref: https://mui.com/api/card/

Reply:
@seblaifa3315 This comment is yet to be addressed too :)